### ISSUE TITLE
SpringRoll Template lacks soft collisions?

### ISSUE THREAD

**Issue by gbhkit** [gopherwood/PlatypusSpringRollTemplate#1]
Following the directions at https://github.com/PBS-KIDS/Platypus/wiki/Build-Instructions, I used this template to make a SpringRoll project. Try as I might, I can't seem to get the portal to trigger collision with the hero. Is it broken? The page at https://github.com/PBS-KIDS/Platypus/wiki/Examples mentions there are no SpringRoll examples. Is there an incompatibility between Platypus and SpringRoll?

UPDATE: It seems to be related to the portal being immobile so it isn't checking for collisions with the hero. Should that be changed so that the hero is checking against the portal instead?

**Comment by probityrules** [comment]
Yes, you're correct. That example was apparently not tested and updated after a collision optimization to reduce the overall number of collision checks.
